### ISSUE TITLE
KDS hash suffixes for DPP sync

### ISSUE THREAD

**Issue by lobkovilya** [kumahq/kuma#8249]
### What happened?

When Zone(Universal) and Global(Kubernetes) we can run into [the same naming collision issue](https://github.com/kumahq/kuma/issues/3837) but for DPPs (because `mesh` is not part of the key on Kubernetes).

This will be fixed by implementing the second step from the [MADR](https://github.com/kumahq/kuma/blob/4f974b720b95b16101aa2f21c196cd3f9d96d430/docs/madr/decisions/029-kds-sync-hash-suffix.md?plain=1#L114-L116):

> Implement Universal labels. This will unblock the use-case with Zone(Universal) and Global(Kubernetes). Also solves the issue with long names on k8s (because we're adding zone+namespace when syncing DPPs). This step also unblocks the ongoing work related to Zone to Global policy sync. When implementing this step make sure Inspect API works in Multizone.
